hips/draw/simple.py

```diff
@@ -178,11 +179,48 @@ def draw_tiles(self) -> np.ndarray:
 
         return image
 
+    def draw_hips_tile_grid(self) -> None:
```

Comment:
I guess. Probably it's not needed then to make it modular and separate?
Maybe just rename this one function and give it a better name and say in the docstring that it does these two things?

@adl1995 - your choice how to finalise this!